copy.go

```diff
@@ -7,9 +7,12 @@ import (
 
 // Configuration struct for redisql
 type Config struct {
+	SQLType     string `yaml:"sqltype"`
 	SQLUser     string `yaml:"sqluser"`
 	SQLPassword string `yaml:"sqlpassword"`
 	SQLDatabase string `yaml:"sqldatabase"`
+	SQLHost     string `yaml:"sqlhost"`
+	SQLPort     string `yaml:"sqlport"`
 	SQLTable    string `yaml:"sqltable"`
 	RedisAddr   string `yaml:"redisaddr"`
 	RedisPass   string `yaml:"redispass"`
```

Comment:
Can you use the `SQLHost` and `SQLPort` parameters in `openSQL` as well? (IIRC the MySQL driver supports this).

Reply:
```openMySQL``` has been changed